### DISCUSSION TITLE
Home Assistant Integration

### DISCUSSION THREAD

**Discussion by mattman73** [awawa-dev/HyperHDR#32] [Q&A] [answered]
hello
Just a quick question, do you have integration support for 'Home Assistant' like Hyperion see [here](https://www.home-assistant.io/integrations/hyperion/)
Many Thanks
Matt

**Comment by awawa-dev** [comment, marked as answer]
Hi
As a fork of Hyperion there is some integration inherited, but it means that it doesn't work well with multiple instances. I made a fix for that in the developing versions and it brings some usability for that (tested with Openhab, maybe have an effect for others apps' integrated using json API).

**Reply by mattman73** [reply]
Thanks
I was not planning on running the two different versions, Once I have HyperHDR running I will test HA intergration.
Thanks
Matt

**Reply by awawa-dev** [reply]
OK. For clarification: instances mean multiple leds controlled simultaneously by one HyperHDR service. For example WS2812b led strip and Philips Hue lamps.
